definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-bailiff-nonprod.json

```diff
@@ -166,5 +166,75 @@
     "CaseStateID": "IssuedToBailiff",
     "UserRole": "[RESPSOLICITOR]",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "3/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "AwaitingAlternativeService",
```

Comment:
what's the purpose of these changes if you're not adding new states?

Reply:
As you have pointed out, these first two StateAuths are redundant, but the rest are required as users must have at least R permissions on a new State